auth0/src/main/java/com/auth0/android/authentication/AuthenticationAPIClient.kt

```diff
@@ -94,18 +93,14 @@ public class AuthenticationAPIClient @VisibleForTesting(otherwise = VisibleForTe
      * Log in a user with email/username and password for a connection/realm.
      * It will use the password-realm grant type for the `/oauth/token` endpoint
      * Example:
-     * <pre>
-     * `client
-     * .login("{username or email}", "{password}", "{database connection name}")
-     * .start(new Callback<Credentials, AuthenticationException>() {
-     * {}Override
-     * public void onSuccess(Credentials payload) { }
-     *
-     * {}Override
-     * public void onFailure(AuthenticationException error) { }
-     * });
-    ` *
-    </pre> *
+     * ```
+     * client
+     *     .login("{username or email}", "{password}", "{database connection name}")
+     *     .start(object : AuthenticationCallback<Credentials> {
```

Comment:
I noticed you swapped the class to `AuthenticationCallback`. Do we want to make these changes now or in the coming weeks?

Reply:
New commit to use `Callback` in docs.